src/Atmos/Model/AtmosModel.jl

```diff
@@ -0,0 +1,164 @@
+module Atmos
+
+export AtmosModel, 
+  ConstantViscosityWithDivergence, 
+  DryModel, MoistEquil,
+  NoRadiation,
+  NoFluxBC, InitStateBC
```

Comment:
```suggestion
  NoFluxBC, ExactStateBC
```
Not really the initial state since it's time dependent. But it is coming from an analytic/exact solution.

Reply:
I've defined it as just calling the initial condition function (which I admit isn't a great name, as it takes a `t` argument).